src/Feature.php

```diff
@@ -13,6 +13,11 @@ class Feature
      */
     private array $globalContext = [];
 
+    /**
+     * @var array<mixed>|null
+     */
+    private ?array $evaluationResults = null;
```

Comment:
**question:** Will this cache take effect considering `app(Feature::class)` in `feature_match` always resolve to a new object?

Reply:
Good point! Let me fix that

Reply:
Fixed!